Code/GraphMol/SmilesParse/SmilesWrite.h

```diff
@@ -76,6 +80,10 @@ enum CXSmilesFields : uint32_t { CXSMILESFIELDS_ENUM_ITEMS };
 RDKIT_SMILESPARSE_EXPORT std::string getCXExtensions(
     const ROMol &mol, std::uint32_t flags = CXSmilesFields::CX_ALL);
 
+//! \brief returns the cxsmiles data for a vector of molecules
+RDKIT_SMILESPARSE_EXPORT std::string getCXExtensions(
+  const MOL_SPTR_VECT &mols, std::uint32_t flags);
```

Comment:
Since `getCXExtensions()` doesn't need to do anything with the ownership of the molecules in the vector, I think it would be better if this took a `std::vector<ROMol *>`.